src/models/index.d.ts

```diff
@@ -1,29 +1,21 @@
 import { ModelInit, MutableModel, PersistentModelConstructor } from "@aws-amplify/datastore";
 
-export enum Difficulty {
-  BEGINNER = "BEGINNER",
+export enum Level {
+  STARTER = "STARTER",
   INTERMEDIATE = "INTERMEDIATE",
-  ADVANCED = "ADVANCED"
+  GURU = "GURU"
 }
 
 export enum Cost {
   ONE = "ONE",
   TWO = "TWO",
-  THREE = "THREE",
-  FOUR = "FOUR"
+  THREE = "THREE"
 }
```

Comment:
i noticed `amplify/backend/questionslist/src/enums.js` calls the pricing levels **`Costs`**. Definitely not a big deal just wanted to draw your attention to that. If it was by design, all good!
<img width="559" alt="Screen Shot 2021-04-11 at 5 18 18 PM" src="https://user-images.githubusercontent.com/37908451/114326513-04e3e100-9aea-11eb-9316-77ff5834e02c.png">

Reply:
damn great catch, that's an inconsistency and I'll change that now. Thanks

Reply:
actually nvm, just realized that's intended since one represents a type (the GraphQL one) and the other represents a map/object that accesses the proper value given the type (JS). Still great eyes though haha

Reply:
haha for sure. sounds good & thx for checking